interview-questions.md

```diff
@@ -7,44 +7,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 
 
 3. Name three rails generator commands. What is created by each?
 
-  Your answer:
+  Your answer: Three rails generator commands are generate model, generate migration, and generate rspec. Generate model creates a database model which can be used to store information, generate migration allows you to modify an existing database, and generate rspec allows you to run tests on your rails code.
 
-  Researched answer:
+  Researched answer: Generate model can be used to create ActiveRecord models. When creating the model you can also generate some fields for it. Generate migration allows you to alter your database schema over time, essentially creating a new version of an existing database. Generate rspec installs the rspec gem, which is a testing framework for a rails application.
 
 
 
 4. Consider the Rails routes below. What is the name of the controller method that would be called by each route? What action would each of the controller methods perform?
 
-action: "GET"    location: /students          
+action: "GET"    location: /students    
+- The controller method is index, and it displays a list of all students.   
 
-action: "POST"   location: /students       
+action: "POST"   location: /students   
+- The controller method is create, it creates a new student entry.    
 
 action: "GET"    location: /students/new
+- The controller method is new, it returns an HTML form for creating a new student entry.
 
-action: "GET"    location: /students/2  
+action: "GET"    location: /students/2
+- The controller method is show, it displays a specific student item.  
 
-action: "GET"    location: /students/2/edit    
+action: "GET"    location: /students/2/edit
+- The controller method is edit, it returns an HTML form for editing a student item.    
 
 action: "PATCH"  location: /students/2      
+- The controller method is update, it updates a specific student item.
 
-action: "DELETE" location: /students/2      
+action: "DELETE" location: /students/2  
+- The controller method is destroy, it deletes a specific student item.
 
 
 
 5. As a developer, you are making an application to manage your to do list. Create 10 user stories that will help you get your application started. Read more about [user stories](https://www.atlassian.com/agile/project-management/user-stories).
+
+- As a user, I can see a web application titled 'My To Do List'
+- As a user, I can see a blank form in list format.
+- As a user, I can add an item to list.
+- As a user, I can check off an item when I complete it.
+- As a user, I can add the time and date that I completed the item to the form.
+- As a user, I can modify a list item.
+- As a user, I can delete an item from the list.
+- As a user, I can create a new list.
+- As a user, I can share my to do list with others.
+- As a user, I can delete my list entirely.
```

Comment:
This is pretty solid, I feel like I have an idea of what you're expecting from a User.